src/components/tasks/Table.js

```diff
@@ -39,7 +39,7 @@ function shouldPSEnabled(_item){
             return (_item.status == status.COMPUTING                        || 
                     _item.status == status.FINISHED                         || 
                     (_item.status == status.TIMEOUT && _item.progress > 0)  ||
-                    (_item.status == status.RESTART && _item.progress > 0))
```

Comment:
If we're holding progress after restart, we need to keep this condition to avoid from opening preview screen without progress.